Sources/RealDeviceMap/Modell/Pokestop.swift

```diff
@@ -57,7 +63,28 @@ class Pokestop: JSONConvertibleObject, WebHookEvent, Hashable {
                 "updated": questTimestamp!,
                 "pokestop_name": name ?? "Unknown",
                 "ar_scan_eligible": arScanEligible ?? 0,
-                "pokestop_url": url ?? ""
+                "pokestop_url": url ?? "",
+                "is_ar": false
```

Comment:
Maybe as part of the global "renaming of a few things" to match the idea that we want to display set A or set B of quests (and not only AR ones of set B), this addition to the webhook sent should be more like `"with_ar": true` here (to reflect this quest is available when holding an AR)?